dagql/idtui/frontend_pretty.go

```diff
@@ -332,50 +298,250 @@ func (fe *frontendPretty) Background(cmd tea.ExecCommand) error {
 	return <-errs
 }
 
+var KeymapStyle = lipgloss.NewStyle().
+	Foreground(lipgloss.ANSIColor(termenv.ANSIBrightBlack))
+	// Background(lipgloss.ANSIColor(termenv.ANSIBlue)).
+	// Foreground(lipgloss.ANSIColor(termenv.ANSIBlack))
+
+func (fe *frontendPretty) renderKeymap(out *termenv.Output, style lipgloss.Style) int {
+	w := new(strings.Builder)
+	type keyHelp struct {
+		label string
+		keys  []string
+		show  bool
+	}
+	var quitMsg string
+	if fe.interrupted {
+		quitMsg = "quit!"
+	} else {
+		quitMsg = "quit"
+	}
+
+	// Blank line prior to keymap
+	for i, key := range []keyHelp{
+		{quitMsg, []string{"q", "ctrl+c"}, true},
+		{out.Hyperlink(fe.cloudURL, "Cloud"), []string{"c"}, fe.cloudURL != ""},
+		{"move", []string{"←↑↓→", "up", "down", "left", "right", "h", "j", "k", "l"}, true},
+		{"first", []string{"home"}, true},
+		{"last", []string{"end", " "}, true},
+		{"zoom", []string{"enter"}, true},
+		{"reset", []string{"esc"}, true},
```

Comment:
incredible nit: I got confused between `reset` and `quit` - `reset` goes to the top-level, and `quit` terminates everything.

I would probably expect `quit` to be at the end of the line as the last option, and maybe call `reset` something else? I can't think of a better name though, so take with a pinch of salt :salt:

Reply:
Done!

* moved quit to the end
* reset -> unzoom, show only when zoomed